training-service/src/app/content/course/component/course-list/course-list.component.html

```diff
@@ -1,17 +1,15 @@
 <div class="container">
-  <div class="row">
-    <div class="form-check" *ngFor="let item of course">
-      <label class="check">
-        <input type="checkbox" (click)="courseClick($event, item.name)">
-          <span class="box">{{ item.name }}</span>
-      </label>
-      <span class="description">
-        opis: <span class="bold">{{ item.description}}</span>,czas trwania: <span class="bold">{{ convertDurationToTime(item.duration)| date:'mm:ss' }} minut</span>,
-        cena <span class="bold">{{ item.price }} zł</span>
-      </span>
-    </div>
+  <div class="list-group">
+    <span *ngFor="let item of course, let i = index">
+    <input type="checkbox" value="{ item.name }" id="checkbox{{i}}" (click)="courseClick($event, item.name)">
```

Comment:
You can use item.name instead of index in checkbox id and label for attr.
Then index will not be needed.

Reply:
API does not return ID of a single record so I used index, added "_" as separator